homework.js

```diff
@@ -0,0 +1,86 @@
+let calculate = function(n){
+    let sum = 0;
+    for (let i = 0; i < n; i++){
```

Comment:
You still want to add the last number `n` to the sum. so your if statement condition like this:
`for (let i = 0; i <= n; i++)`